Translation files updated to work with Django 1.4, footer translated

premik91 opened the pull request:
Footer translated, documentation updated.

I remember reading in one of your comments in other requests, that we shouldn't use django-admin.py, but since manage.py was moved, frendo's "tutorial" didn't work anymore, and I found this working replacement. I also hope that this is not another of my multi-feature problems. I think all of this belongs to translation feature.

Commit:
Translation files updated to work with Django 1.4, footer translated

Footer translated, documentation updated

mitar commented on docs/translation.rst:
```diff
@@ -4,36 +4,37 @@ Translation of PiplMesh
 Creation of translation files
 -----------------------------
    
 .. warning:: It is important to run folowing commands from repository root directory!
```
In fact this is not correct. The proper way is still to run things from project directory, but to run it as:

```
../manage.py makemessages -l <language code>
```

See two dots.

premik91 replied:
Oh I tried everything and went through django documentation and haven't found anything but django-admin.py. I didn't even try the simplest solution. Sorry for my sloppiness.

mitar commented on piplmesh/frontend/templates/footer.html:
```diff
@@ -2,26 +2,29 @@
     <hr class="hidden" />
     <div id="footer_nav">
-        <a href="http://wlan-si.net/">wlan slovenija</a> | <a href="https://nodes.wlan-si.net/">node database</a> | <a href="http://dev.wlan-si.net/">development</a> | <a href="http://grow.wlan-si.net/">growing</a>
+        <a href="http://wlan-si.net/">{% trans "wlan slovenija" %}</a> |
```
_wlan slovenija_ is name, this is not translated.

premik91 replied:
I didn't translate it, I just (don't know why) added it as a translatable.

mitar commented on piplmesh/frontend/templates/footer.html:
```diff
@@ -2,26 +2,29 @@
-                {% endif %}
+            {% get_language_info for lang.0 as lang_info %}
+            {% if lang.0 == LANGUAGE_CODE %}
+            <li class="language_selected">{{ lang_info.name_local }}</li>
```
Plese indent all in the same way as it was before (and block/tag, template or HTML, is indented).

mitar replied:
I am not sure what have you changed here except indentation?

premik91 replied:
I noticed now, PyCharm did the indetation, I changed only the 'footer_nav', added translation tags and moved them each to new line(I am just used to it, beacause it is easier to read, and as far as I know it does no harm). I will of course put it back and never do it again

mitar replied:
Splitting footer into multiple lines is probably a good thing, jut don't break indentation at the same time. :-)

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -5,8 +5,8 @@
 "Project-Id-Version: piplmesh\n"
-"Report-Msgid-Bugs-To: https://github.com/mitar/PiplMesh\n"
-"POT-Creation-Date: 2012-03-25 20:05+0200\n"
+"Report-Msgid-Bugs-To: \n"
```
Please make sure URL is kept (officially it should be e-mail address). But update it to new location.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
-#: account/forms.py:14
-msgid "Male"
+#: account/fields.py:9
+#, fuzzy
```
Remove those comments when you are reviewing translations. Always check the file.

premik91 replied:
Yes I noticed them, but wasn't sure if I should delete them

mitar replied:
I am deleting it. As far I understand it just tells you that this was automatically derived from some similar translation. And that you should check it. But you can check what Internet says about this.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
+#: account/fields.py:9
+#, fuzzy
+msgid "male"
 msgstr "Moški"
```
Case is obviously wrong. Fix.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
-msgstr "Uporabniško ime"
+#: account/form_fields.py:11
+msgid "Value is out of bounds."
+msgstr "Vrednost je izven dovoljenega."
```
_Vrednost je izven meja._

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
+#: account/signals.py:11
+#, fuzzy
+msgid "You have been successfully logged in."
+msgstr "Odjava uspešna."
```
_Prijava uspešna._

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
 
-#: account/templates/login.html:5 account/templates/login.html.py:14
+#: account/signals.py:11
+#, fuzzy
```
Same.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
 
-#: frontend/templates/header.html:35
+#: frontend/templates/header.html:43
 msgid "Log in"
```
Fix this to be always and everywhere _Login_ (one word).

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
 
-#: frontend/templates/header.html:32
+#: frontend/templates/header.html:38
 msgid "Log out"
```
One word.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
 
-#: account/templates/login.html:19
+#: account/templates/registration/login.html:25
 msgid "Log in with Facebook"
```
_Login_

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
 msgid "Reset"
 msgstr "Ponastavi"
 
+#: frontend/templates/footer.html:5
```
Remove.

mitar commented on piplmesh/locale/sl/LC_MESSAGES/django.po:
```diff
@@ -16,103 +16,144 @@ msgstr ""
+msgid "Search"
+msgstr "Išči"
+
+#~ msgid "Male"
```
For those string, check why they are not needed anymore and if there is no error (like somebody forgot to mark strings as translatable), remove them.

Commit:
Fixed

Fuzzy lines deleted and translation fixed,
updated docs/translation.rst,
fixed 'log in' and 'log out' to one word (also in comment)

Commit:
Fixed

docs/translation.rst

Commit:
Added missing translations

premik91 commented:
I am very sorry for my sloppiness and your troubles yet again(something here brings the worst in me). The thing was, I just wanted to update and add things, so I wasn't paying enough attention to what has changed since the last translation(and many things did). I have also searched for the missing translations and haven't been able to find them in the code (but they are certainly there on registration forms):

#~ msgid "Username"
#~ msgstr "Uporabniško ime"

#~ msgid "Password"
#~ msgstr "Geslo"

#~ msgid "Repeat password"
#~ msgstr "Ponovi geslo"

#~ msgid "Username:"
#~ msgstr "Uporabniško ime:"

#~ msgid "Password:"
#~ msgstr "Geslo:"

mitar commented:
> (something here brings the worst in me).

What, you have a bottle of vine next to you?

premik91 commented:
Nope, just a latop and reading what stupid mistakes I did. No bottle needed, it is sad enough :)

mitar commented:
> I have also searched for the missing translations and haven't been able to find them in the code (but they are certainly there on registration forms):

Don't worry about those, they are not in our code, but official Django distribution and are already translated there. So you can remove them.

Commit:
Removed past translations

premik91 commented:
Fixed

mitar commented:
Good work and great self-initiative! Now we have translations again current.
